app/core/settings.py

```diff
@@ -1,9 +1,12 @@
 import os
+import dj_database_url
 
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-SECRET_KEY = 'bi6y(ew+jh54zu-gs#yete^c13_8f##jgq3lj9s&nfg-6!06zn'
-DEBUG = True
-ALLOWED_HOSTS = '*'
+ALLOWED_HOSTS = os.environ.get('ALLOWED_HOSTS', 'localhost').split(',')
+SECRET_KEY = os.environ.get('APP_SECRET', 'secret')
+
+ENVIRONMENT = os.environ.get('APP_ENV', 'dev')
+DEBUG = ENVIRONMENT in ['ci', 'dev']
```

Comment:
I'd favor an explicit `DEBUG` environment variable to select, so that we can turn it on/off depending on the environment and the use case. Then `APP_ENV` can disappear